scripts/screeps_rl_live_dashboard.py

```
@@ -73,12 +86,74 @@ def __init__(
+            return dict(self.refresh_state)
+
+    def start_auto_refresh(self) -> None:
+        if self.refresh_thread is not None:
+            return
+
+        def refresh_loop() -> None:
+            while not self.refresh_stop.wait(self.config.auto_refresh_seconds):
+                with self.refresh_lock:
+                    try:
+                        refresh = refresh_metrics(self.config.db_path, self.config.artifact_root)
+                    except Exception as error:  # pragma: no cover - defensive background loop boundary
```

Ruff (0.15.13)

[warning] Do not catch blind exception: `Exception`

(BLE001)